Packs/FeedMitreAttack/Misc/indicator.json

```diff
@@ -0,0 +1,32 @@
+[
+	{
```

Comment:
please change file name to `reputation-mitreAttck` and move it to `Misc` dir outside the pack

Reply:
There is no directory named "Misc" outside of the pack??

Reply:
Done.